### ISSUE TITLE
Rank Hierarchy via Signs?

### ISSUE THREAD

**Issue by FrostBlade5** [ElgarL/GroupManager#83]
I would like to create a rank hierarchy where players get the default rank when first joining and earn money for killing mobs, once they have enough money they will be able to buy the next rank. important, not just buy any rank of choice but buy the next one in order. So players with more time on the server will benefit from using rankup or promotion signs.

Is this possible with groupmanager / essentials ? Or do i need an "addon" ?

Basically buy all the ranks one by one in specific order without skipping any of them.

**Comment by ElgarL** [comment]
Sounds like you need something like https://www.spigotmc.org/resources/signshop.10997/
 Its probably possible with the Essentials signs too, but its been many years since I used Essentials.
